pom.xml

```diff
@@ -170,7 +169,11 @@
                     </execution>
                 </executions>
             </plugin>
-
+            <plugin>
+                <groupId>org.sonarsource.scanner.maven</groupId>
```

Comment:
the plugin is not enabled by default?

Because everything that makes the local build slower is a PITA

On Travis, that perfectly makes sense to run all these sentity checks for code smells etc

Reply:
Correct right now it's just automated via Travis. I deliberately excluded the local version (which is just additional prop files included) to keep the build time sane. 

I intend on a 2nd PR with instructions for a dev to run it locally if they want, given that the plugin is there it would be a few additional steps on the dev to get Sonar compatibility.